persistence-modules/mybatis-vs-hibernate/src/main/java/com/baeldung/mybatisvshibernate/Student.java

```diff
@@ -0,0 +1,88 @@
+package com.baeldung.mybatisvshibernate;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+
+import java.util.Objects;
+
+@Entity
+public class Student {
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @Column
+    private String name;
+
+    @Column
+    private int phone;
+
+    @Column
+    private String email;
+
+    public Student() {
+    }
```

Comment:
If this is required for Hibernate, make it clear with a comment
```suggestion
    public Student() {
        // required for Hibernate
    }
```